src/main/java/core/basesyntax/strategy/OperationHandler.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax.strategy;
+
+public interface OperationHandler {
+    void calculateValue(String fruit, int quantity);
```

Comment:
Since we should cover this code with test I would recommend to return some value like new HashMap<String,Integer> after changes